migration-from-4x.md

```diff
@@ -417,10 +431,12 @@ tinymce.init({
     help: { title: 'Help', items: 'help | myCustomMenuItem' }
   },
   menubar: 'file edit help',
-  setup: (editor) => {
+  setup: function (editor) {
     editor.ui.registry.addMenuItem('myCustomMenuItem', {
       text: 'My Custom Menu Item',
-      onAction: () => alert('Menu item clicked')
+      onAction: function () {
+        alert('Menu item clicked')
```

Comment:
Missing semicolon at the end of the line here.

Reply:
Fixed in https://github.com/tinymce/tinymce-docs/pull/1645/commits/50d665fdbcdb9afd3395dccf0ff6eba409f4895d